scipy/integrate/_quadrature.py

```diff
@@ -376,6 +388,70 @@ def quadrature(func, a, b, args=(), tol=1.49e-8, rtol=1.49e-8, maxiter=50,
     >>> np.sin(np.pi/2)-np.sin(0)  # analytical result
     1.0
 
+    """
+    return gauss_quad(func, a, b, args, tol, rtol, maxiter, vec_func, miniter)
+
+
+def gauss_quad(func, a, b, args=(), tol=1.49e-8, rtol=1.49e-8, maxiter=50,
+               vec_func=True, miniter=1):
+    r"""
+    Compute a definite integral using fixed-tolerance Gaussian quadrature.
+
+    .. legacy:: function
+
+        This function is considered legacy as of SciPy 1.12.0; it may
+        be deprecated and removed in the future. Please use
+        `scipy.integrate.quad` for new code.
```

Comment:
This was a bit redundant:
![image](https://github.com/scipy/scipy/assets/6570539/f5be8824-484c-4968-a7da-1e87f3b4dfd4)

so:

```suggestion
        Please use `scipy.integrate.quad` for new code.
```